style.css

```diff
@@ -0,0 +1,815 @@
+	margin-bottom: 4px;
+}
+
+.header__language {
+	left: 29.5%;
+	top: 34px;
+	font-size: 16px;
+}
+
+.header__vector {
+	left: 32.5%;
+	top: 42px;
+}
+
+.img__img {
+	width: 439px;
+	height: 450px;
+}
+
+.step {
+	position: absolute;
+	width: 100%;
+	display: grid;
+	grid-template-columns: 439px 439px;
+	grid-template-rows: 450px;
+	justify-content: center;
+	top: 135px;
+	z-index: 2;
+	grid-column-gap: 42px;
+}
+
+.text {
+	height: 450px;
+}
+
+.text__title {
+	font-size: 60px;
+	line-height: 68px;
+	letter-spacing: 4px;
+	width: 439px;
+}
+
+.text__container {
+	padding: 39px 0;
+}
+
+.text__subtitle {
+	font-size: 12px;
+	line-height: 16px;
+	letter-spacing: 1.1px;
+	margin-bottom: 40px;
+}
+
+.collection__next-text {
+	font-size: 11px;
+	padding: 10px 27px;
+	margin-right: 43px;
+	letter-spacing: 0px;
+}
+
+.collection__other-text {
+	font-size: 11px;
+	letter-spacing: 0px;
+	margin-right: 15px;
+}
+
+.collection__vector {
+	width: 35px;
+}
+
+.collection__scroll {
+	width: 22px;
+	height: 39px;
+	left: 46.5%;
+	top: 650px;
+}
+
+.collection__scroll-elem {
+	left: 47.2%;
+	top: 659.5px;
+}
+
+.first__sidebar-social-inst, 
+.first__sidebar-social-vk, 
+.first__sidebar-social-fb {
+	width: 35px;
+	height: 35px;
+}
+
+.first__sidebar-social {
+	width: 123px;
+	padding: 652px 45px 0;
+}
+}
+
+@media (max-width: 1000px) {
+.first {
+	max-width: 1000px;
+	width: 100%;
+	display: grid;
+	grid-template-columns: 35.2% 64.8%;
+	grid-template-rows: 541px;
+}
+
+.header__logo {
+	font-size: 17px;
+	letter-spacing: 2px;
+	margin: 4px 31px;
+}
+
+.header__menu {
+	height: 54px;
+	width: 54px;
+}
+
+.header__menu-item {
+	height: 3px;
+	width: 22.7px;
+	margin-bottom: 4px;
+}
+
+.header__language {
+	left: 29.5%;
+	top: 24px;
+	font-size: 12px;
+}
+
+.header__vector {
+	left: 32%;
+	top: 31px;
+	width: 9px;
+	height: 6px;
+}
+
+.img__img {
+	width: 329px;
+	height: 338px;
+}
+
+.step {
+	position: absolute;
+	width: 100%;
+	display: grid;
+	grid-template-columns: 329px 329px;
+	grid-template-rows: 338px;
+	justify-content: center;
+	top: 101px;
+	z-index: 2;
+	grid-column-gap: 31px;
+}
+
+.text {
+	width: 329px;
+	height: 338px;
+}
+
+.text__title {
+	font-size: 49px;
+	line-height: 50px;
+	letter-spacing: 1px;
+	width: 329px;
+}
+
+.text__container {
+	padding: 27px 0;
+}
+
+.text__subtitle {
+	font-size: 9px;
+	line-height: 13px;
+	letter-spacing: 1px;
+	margin-bottom: 24px;
+}
+
+.collection__next-text {
+	font-size: 8px;
+	padding: 7px 22px;
+	margin-right: 31px;
+	letter-spacing: 0px;
+}
+
+.collection__other-text {
+	font-size: 8px;
+	letter-spacing: 0px;
+	margin-right: 3px;
+}
+
+.collection__vector {
+	width: 27px;
+}
+
+.collection__scroll {
+	width: 17px;
+	height: 28px;
+	left: 46.5%;
+	top: 489px;
+}
+.collection__scroll-elem {
+	left: 47.3%;
+	top: 494.5px;
+	width: 2px;
+}
+
+.first__sidebar-social-inst, 
+.first__sidebar-social-vk, 
+.first__sidebar-social-fb {
+	width: 25px;
+	height: 25px;
+}
+
+.first__sidebar-social {
+	width: 92px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)